modules/op-custom-parameter-expansion.adoc

```diff
@@ -0,0 +1,69 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: CONCEPT
+[id="op-custom-parameter-expansion_{context}"]
+= Custom parameter expansion
+
+{pac} helps you expanding a custom parameter within your `PipelineRun` resource by using the `params` field. You can specify a value for the custom parameter inside the template of the `Repository` CR. The specified value replaces the custom parameter in your pipeline run.
+
+[NOTE]
+====
+Use the custom parameter expansion feature only in specific scenarios when you cannot use the Tekton `PipelineRun` parameters.
```

Comment:
@chmouel @savitaashture why it should be used only in specific scenarios?

Reply:
@ppitonak Please approve the PR if there are no further comments.